payments-core/src/main/java/com/stripe/android/view/CardFormView.kt

```diff
@@ -49,7 +49,7 @@ class CardFormView @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
-) : LinearLayout(context, attrs, defStyleAttr) {
+) : ComposeLifecycleOwner(context, attrs, defStyleAttr) {
```

Comment:
Is this still needed now or is this a remnant from an earlier approach?

Reply:
Yeah it's still needed to access the viewmodel in Dialog/React Native contexts